plugin.yaml

```diff
@@ -69,3 +226,92 @@ relationships:
   cloudify.gcp.relationships.contained_in:
     derived_from: cloudify.relationships.contained_in
 
+  cloudify.gcp.relationships.agent_connected_to_security_group:
```

Comment:
You're right. I was more concerned on getting manager blueprint working that the design itself.

Tag is used for security groups that are based on firewall rules. They differ from what you can find in Openstack and AWS so I had to create something that will behave the same way. I could explain you that on Skype.